spec/02-integration/09-hybrid_mode/09-node-id-persistence_spec.lua

```diff
@@ -94,6 +94,7 @@ for _, strategy in helpers.each_strategy() do
       cluster_cert_key = "spec/fixtures/kong_clustering.key",
       cluster_listen = "127.0.0.1:9005",
       nginx_conf = "spec/fixtures/custom_nginx.template",
+      cluster_rpc = "on",
```

Comment:
should we set it to inc_sync?

Reply:
Not sure about it, but I think that cp should always support two prorocols.

Reply:
i know andy's meaning, but it need some effort to enable it, he want us to test the following 3 kinds of scenarios:

1. rpc = off, inc_sync = off <-- and this one is our default action, this is not tested in these test cases
2. rpc = on, inc_sync = off
3. rcp = on, inc_sync = on

Reply:
done, thank you.